### DISCUSSION TITLE
Suggestion: Links for each user

### DISCUSSION THREAD

**Discussion by mohamm6d** [ash-jc-allen/short-url#118] [Ideas]
First of all, this package is awesome, I just found it. Thank you for that.
As a suggestion, would it be possible to add another new column to the migration to store user_id, so it would be multi-user feature :)

**Comment by ash-jc-allen** [comment]
Hey @mohamm6d! Thanks, I'm really glad that you like the package 😄

Adding a `user_id` column is definitely something that I could see being useful in some people's applications, but not necessarily everyone's. So, I don't think it's something that I'd add to the migrations that are provided with the package.

But if anyone wants to add the `user_id` column themselves, they could edit the migration (that's published) and add the column themselves. Hopefully this helps out a bit? :)

**Reply by mohamm6d** [reply]
Hey. Thank you for your reply. Yes, this is what I did actually :D
